### PR TITLE
[HWORKS-1113] Add docs for attaching model evaluation images to a model

### PR DESCRIPTION
**Pull request by javierdlrm**

### CODE REVIEW

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# How To Save Model Evaluation Images
```
Comment (SirOibaf) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:1]: add description tag here

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+description: Documentation on how to attach model evaluation images to a model.
+---
+
+# How To Save Model Evaluation Images
+
+## Introduction
+
+In this guide, you will learn how to attach ==model evaluation images== to a model. Model evaluation images contain **confusion matrices**, **ROC curves** or other graphs that help visualizing model evaluation metrics. By attaching model evaluation images to your model version, other users can better understand the experiment results obtained during model training.
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:9]: Model evaluation images are images that visually  describe model performance metrics. For example, **confusion matrices**, **ROC curves**, **model bias tests**, and **training loss curves** are examples of common model evaluation images.

By attaching model evaluation images to your versioned model

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+```python
+import hopsworks
+
+project = hopsworks.login()
+
+# get Hopsworks Model Registry handle
+mr = project.get_model_registry()
+```
+
+### Step 2: Generate model evaluation figures
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:24]: images

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Generate a figure that visualizes a model metric.
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:26]: an image

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+heatmap = seaborn.heatmap(df_confusion_matrix, annot=True)
+
+# Get the figure and display it
+fig = heatmap.get_figure()
+fig.show()
+```
+
+### Step 3: Save the figures as images inside the model directory
+
+Save the figure to an image file, and place it in a directory with name ´images´ inside the model directory to be exported.
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:58]: Save the figure to an image file -> Save the figure to file with a common filename extension (for example, .png or .jpeg), and place it in a directory called ´images´ - a subdirectory of the model directory that is registered to Hopsworks.

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+```python
+# Specify the directory name for saving the model and related artifacts
+model_dir = "./model"
+
+# Create a directory with name 'images' for saving the model evaluation images
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:64]: Create a subdirectory of `model_dir` called 'images' for saving the model evaluation images

File: docs/user_guides/mlops/registry/model_evaluation_images.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+fig.savefig(model_images_dir + "/confusion_matrix.png")
+
+# Register the model
+py_model = mr.python.create_model(name="py_model")
+py_model.save("./model")
+```
+
+## Conclusion
+
+In this guide you learned how to attach model evaluation images to a model, helping better understand the experiment results obtained during model training.
```
Comment (jimdowling) [inline docs/user_guides/mlops/registry/model_evaluation_images.md:79]: the experiment results obtained during model training. ->
model performance and evaluation metrics .

Comment (jimdowling) [reply docs/user_guides/mlops/registry/model_evaluation_images.md:79]: helping better understand the experiment results  ->
 visually communicating the model performance and evaluation metrics in the model registry.

**Review by jimdowling — APPROVED** [review]
Some re-formulating of text

**Pull request merged**